conf/app.routes

```diff
@@ -7,8 +7,7 @@ POST 	    /fraud
 POST        /rds/assessments/self-assessment-assist                                       controllers.RdsController.generateReport()
 POST        /rds/assessments/self-assessment-assist/acknowledge                           controllers.RdsController.acknowledgeReport()
 
-POST 	    /individuals/self-assessment-assist/submission/report                         controllers.NrsController.submit()
-
+POST 	    /submission                                                                   controllers.NrsController.submit()
```

Comment:
Please make sure we have updated application.conf in our api with this path, as well as app-config-base